apps/extension/src/components/InfoBox/InfoBox.tsx

```
@@ -0,0 +1,34 @@
+import React from 'react';
+
+interface InfoBoxProps {
+  version?: Number;
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)